### DISCUSSION TITLE
博士屯和大农村 | /cn/2024-07-12-/

### DISCUSSION THREAD

**Discussion by giscus** [JinjiPang/jinji#56] [Comments]
# cn/2024-07-12-/

不知不觉，我在博士屯已经生活了九个月了。初识在深秋，忧郁在严冬，憎恨在初春，夏季的到来，让我渐渐开始适应和跟上了这里的生活节奏，甚至有些喜欢。
不知是不是命运轮回，来自中国东北五线小城的我，毕业后因为人生中第一份工作，搬到了漂亮国的东北。虽然有着相似的气候，四季分明，长达半年的冬天，但是此东北非彼东北，一个是没落的共和国长子，一个则是美国历史最悠久，至今也是人均受教育程度最高、最富裕的新英格兰地 &hellip;

https://jinjipang.com/cn/2024-07-12-/

**Comment by WasmachstduRudiger** [comment]
这句话很好
